boards/arm/mimxrt595_evk/board.c

```diff
@@ -98,9 +98,31 @@ static int mimxrt595_evk_init(void)
 	return 0;
 }
 
+
+#ifdef CONFIG_LV_Z_VBD_CUSTOM_SECTION
+extern char __flexspi2_start[];
+extern char __flexspi2_end[];
+
+static int init_lvgl_bufs(void)
+{
+	/*
+	 * LVGL rendering buffers will be stored in FlexSPI2 linker section.
+	 * Zero out BSS section.
+	 */
+	memset(__flexspi2_start, 0, __flexspi2_end - __flexspi2_start);
+	return 0;
+}
+
+#endif /* CONFIG_LV_Z_VBD_CUSTOM_SECTION */
+
 #if CONFIG_REGULATOR
 /* PMIC setup is dependent on the regulator API */
 SYS_INIT(board_config_pmic, POST_KERNEL, CONFIG_APPLICATION_INIT_PRIORITY);
 #endif
 
+#ifdef CONFIG_LV_Z_VBD_CUSTOM_SECTION
+/* LVGL buffers should be setup after PSRAM is initialized but before LVGL init */
+SYS_INIT(init_lvgl_bufs, POST_KERNEL, CONFIG_APPLICATION_INIT_PRIORITY);
+#endif
+
```

Comment:
That does not look right to me. There should be no third-party project-specific code in the boards/soc area. It could be dc_ram/dc_ram.ld, which would be used by any display subsystem.

Reply:
Ok, I have renamed the linker file to `dc_ram.ld`, and the function call to `init_psram_framebufs`